smdebug/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.5.0"
+__version__ = "0.5.0.post0"
```

Comment:
nit: new line

why this change? in the nightly pre-build release, we fetch this version number append 'b' 'timestamp' to generate the pre-build version. this upload upload might now fail since `0.5.0.post0b<timestamp>` might not meet semver standards of pypi.

Reply:
this was because 0.5.0 publish happened without description for pypi so next version after that is 0.5.0.post0.